src/interface/common.thrift

```diff
@@ -403,7 +403,7 @@ enum ErrorCode {
     E_INVALID_JOB                     = -2065,  // Invalid task
 
     // Backup Failure
-    E_BACKUP_BUILDING_INDEX           = -2066,  // Backup terminated (index being created)
+    E_BACKUP_RUNNING_JOBS             = -2066,  // Backup terminated (index being created)
```

Comment:
plz update the error code comment and revisit the compatible issue caused by this thrift change

Reply:
good catch, thanks.